packages/desktop/views/dashboard/account-management/AccountManagement.svelte

```diff
@@ -26,17 +17,17 @@
 
     function formatAndTruncateAccount(output) {
         let address: string = ''
-        const implicitUnlockCondition = (output as CommonOutput).unlockConditions.find(
-            (output) => output.type === UnlockConditionType.Address
-        ) as AddressUnlockCondition
-
-        if ((output as AccountOutput).accountId) {
-            address = getBech32AddressFromAddressTypes(new AccountAddress((output as AccountOutput).accountId))
-        } else if (implicitUnlockCondition.address.type === AddressType.ImplicitAccountCreation) {
+        if (isImplicitAccountOutput(output)) {
```

Comment:
something is wrong here because the address is not displayed correctly
![Captura de pantalla de 2024-02-13 09-16-13](https://github.com/iotaledger/firefly/assets/100352899/85513164-cb4a-46cb-9391-019440016836)

Reply:
maybe you have to pull because they're displaying correctly for me... :see_no_evil: 
![image](https://github.com/iotaledger/firefly/assets/72078237/40f997d9-16c2-4e80-b707-e566d2aad6b6)